Samples/SendRestExample/SendRestExample/ConnectionStringUtility.cs

```diff
@@ -15,26 +15,34 @@ public ConnectionStringUtility(string connectionString)
             //Parse Connectionstring
             char[] separator = { ';' };
             string[] parts = connectionString.Split(separator);
-            for (int i = 0; i < parts.Length; i++)
+            foreach (var t in parts)
             {
-                if (parts[i].StartsWith("Endpoint"))
-                    Endpoint = "https" + parts[i].Substring(11);
-                if (parts[i].StartsWith("SharedAccessKeyName"))
-                    SasKeyName = parts[i].Substring(20);
-                if (parts[i].StartsWith("SharedAccessKey"))
-                    SasKeyValue = parts[i].Substring(16);
+                if (t.StartsWith("Endpoint"))
+                {
+                    Endpoint = "https" + t.Substring(11);
+                }
+
+                if (t.StartsWith("SharedAccessKeyName"))
+                {
+                    SasKeyName = t.Substring(20);
+                }
+
+                if (t.StartsWith("SharedAccessKey"))
+                {
+                    SasKeyValue = t.Substring(16);
+                }
             }
         }
 
-        public string getSaSToken(string uri, int minUntilExpire)
+        public string GetSaSToken(string uri, int minUntilExpire)
         {
             string targetUri = Uri.EscapeDataString(uri.ToLower()).ToLower();
 
             // Add an expiration in seconds to it.
             long expiresOnDate = DateTime.Now.Ticks / TimeSpan.TicksPerMillisecond;
             expiresOnDate += minUntilExpire * 60 * 1000;
-            long expires_seconds = expiresOnDate / 1000;
-            String toSign = targetUri + "\n" + expires_seconds;
+            long expiresSeconds = expiresOnDate / 1000;
+            String toSign = targetUri + "\n" + expiresSeconds;
```

Comment:
Let's update this method to use `var` for the variables. The types are obvious from the usage.